2D Simulation/blocks.js

```diff
@@ -0,0 +1,55 @@
+/* All references to id's are just a number. No strings allowed! */
+let blocks = new function () {
+    function get_block(id) {
```

Comment:
Does the difference with lifting matter here or could this be changed to be in the same style as the other function declarations?